15/umbraco-commerce/release-notes/v15.0.0-rc.md

```diff
@@ -0,0 +1,83 @@
+---
+description: >-
+  Umbraco Commerce v15.0.0-rc release notes.
+---
+
+# Umbraco Commerce v15.0.0-rc
+
+Umbraco Commerce v15.0.0-rc is the initial release of Umbraco Commerce for Umbraco CMS v15. Because of 
+
+## Key Takeaways
+
+* Everything is now [async](#async).
+* [Storefront API](#storefront-api) aligned with Management API
+* A number of [Umbraco v15 updates](#umbraco-v15-updates).
+
+## Async
+
+The key focus of this release is the move to a fully asyncronus code base. In order to reduce the maintenance burden, the decision was made to go fully async without maintaining backwards compatibility. This will therfore require code updates to use the new async methods.
```

Comment:
```suggestion
The key focus of this release is the move to a fully asynchronous code base. To reduce the maintenance burden, the decision was made to go fully async without maintaining backward compatibility. This will therefore require code updates to use the new async methods.
```